src/execution/operator/aggregate/physical_ungrouped_aggregate.cpp

```diff
@@ -62,8 +204,25 @@ struct AggregateState {
 
 class SimpleAggregateGlobalState : public GlobalSinkState {
```

Comment:
Nitpick: Can we rename anything that is still called `SimpleAggregate...` to `UngroupedAggregate...`?

The operator was recently renamed, and I think its states should be renamed as well.